packages/analytics/dashboard-renderer/src/locales/en.json

```diff
@@ -0,0 +1,6 @@
+{
+  "header": {
+    "title": "Title of the header",
+    "content": "Content of the header"
+  }
+}
```

Comment:
You can substitute your own strings here and remove this example content

Reply:
Done.